.clang-format

```
@@ -0,0 +1,148 @@
+---
+# this file work is a derivative of .clang-format which follows
+# https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md
+#
+Language:        Cpp
+BasedOnStyle:  Microsoft
+AccessModifierOffset: -4
+AlignAfterOpenBracket: AlwaysBreak
+AlignConsecutiveAssignments: false
+AlignEscapedNewlines: DontAlign
+AlignOperands:   true
+AlignTrailingComments: true
+AllowAllParametersOfDeclarationOnNextLine: true
+AllowShortBlocksOnASingleLine: false
```

what version of `clang-format` is this based on? the [docs](https://clang.llvm.org/docs/ClangFormatStyleOptions.html) list this as an enum value of `Never`, `Empty`, and `Always`...

---

I believe the old option is true/false because I found `false` in other clang-format too
```suggestion
AllowShortBlocksOnASingleLine: Never
```